osidb/migrations/0001_initial_squashed_0059_flaw__alerts.py

```diff
@@ -0,0 +1,1494 @@
+                        to="osidb.packageversions",
+                    ),
+                ),
+                ("package", models.CharField(max_length=2058)),
+                (
+                    "default_status",
+                    models.CharField(
+                        choices=[
+                            ("AFFECTED", "Affected"),
+                            ("UNAFFECTED", "Unaffected"),
+                            ("UNKNOWN", "Unknown"),
+                        ],
+                        default="UNAFFECTED",
+                        max_length=1024,
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+                "base_manager_name": "objects",
+            },
+            bases=("osidb.packageversions",),
+        ),
+        migrations.CreateModel(
+            name="CVEv5Version",
+            fields=[
+                (
+                    "version_ptr",
+                    models.OneToOneField(
+                        auto_created=True,
+                        on_delete=django.db.models.deletion.CASCADE,
+                        parent_link=True,
+                        primary_key=True,
+                        serialize=False,
+                        to="osidb.version",
+                    ),
+                ),
+                ("version", models.CharField(max_length=1024)),
+                (
+                    "status",
+                    models.CharField(
+                        choices=[
+                            ("AFFECTED", "Affected"),
+                            ("UNAFFECTED", "Unaffected"),
+                            ("UNKNOWN", "Unknown"),
+                        ],
+                        max_length=20,
+                    ),
+                ),
+            ],
+            options={
+                "abstract": False,
+                "base_manager_name": "objects",
+            },
+            bases=("osidb.version",),
+        ),
+        ###########
+        # PROFILE #
+        ###########
+        migrations.CreateModel(
+            name="Profile",
+            fields=[
+                (
+                    "user",
+                    models.OneToOneField(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        primary_key=True,
+                        related_name="profile",
+                        serialize=False,
+                        to="auth.user",
+                    ),
+                ),
+                (
+                    "bz_user_id",
+                    models.CharField(blank=True, max_length=100),
+                ),
+                (
+                    "jira_user_id",
+                    models.CharField(blank=True, max_length=100),
+                ),
+            ],
+        ),
+        ######################
+        # ROW LEVEL SECURITY #
+        ######################
+        migrations.RunSQL(
```

Comment:
Just a thought: since this SQL setting the RLS is irreversible the whole migration would be irreversible too which might not be convenient when playing around with migrations for whatever reason. Shouldn't we make this reversible? It can be done by supplying the argument `reverse_sql=migrations.RunSQL.noop` to `migrations.RunSQL`.

Reply:
Good point! I will make add that reverse noop